src/rpc/client.cpp

```diff
@@ -299,7 +299,7 @@ UniValue RPCConvertNamedValues(const std::string &strMethod, const std::vector<s
     }
 
     if (!positional_args.empty()) {
-        params.pushKV("args", positional_args);
+        params.__pushKV("args", positional_args);
```

Comment:
763d3947d34fa71f1a0816c95fb6e206507e2ece

Using` args=` is fine but not when used along with positional arguments. Agree with @stickies-v, it's not that clear from the error message what the user should do.

An alternative is to fail here if `params` already contain `args`.